src/pages/episode/sections/show-notes/index.js

```diff
@@ -16,9 +16,9 @@ function ShowNotes({episode}) {
     return <noscript />
   }
   return (
-    <section id="show-notes">
```

Comment:
The ID should stay so we can hard-link to the show notes section of the page :+1: (for example: https://javascriptair.com/episodes/2016-06-15/#show-notes)